tests/bluetooth/audio/ascs/src/test_ase_control_params.c

```diff
@@ -0,0 +1,953 @@
+		0x01,           /* Number_of_ASEs */
+		0x01,           /* ASE_ID[0] */
+		0x01,           /* Target_Latency[0] */
+		target_phy,     /* Target_PHY[0] */
+		0x06,           /* Codec_ID[0].Coding_Format = LC3 */
+		0x00, 0x00,     /* Codec_ID[0].Company_ID */
+		0x00, 0x00,     /* Codec_ID[0].Vendor_Specific_Codec_ID */
+		0x00,           /* Codec_Specific_Configuration_Length[0] */
+	};
+	const uint8_t data_expected[] = {
+		0x01,           /* Opcode */
+		0x01,           /* Number_of_ASEs */
+		0x01,           /* ASE_ID[0] */
+		0x00,           /* Response_Code[0] = Success */
+		0x00,           /* Reason[0] */
+	};
+
+	fixture->ase_cp->write(&fixture->conn, fixture->ase_cp, buf, sizeof(buf), 0, 0);
+
+	expect_bt_gatt_notify_cb_called_once(&fixture->conn, BT_UUID_ASCS_ASE_CP,
+					     fixture->ase_cp, data_expected, sizeof(data_expected));
+}
+
+ZTEST_F(test_ase_control_params, test_target_phy_out_of_range_0x00)
+{
+	/* TODO: Remove once resolved */
+	Z_TEST_SKIP_IFNDEF(BUG_55794);
+
+	test_target_phy_out_of_range(fixture, 0x00);
+}
+
+ZTEST_F(test_ase_control_params, test_target_phy_out_of_range_0x04)
+{
+	/* TODO: Remove once resolved */
+	Z_TEST_SKIP_IFNDEF(BUG_55794);
+
+	test_target_phy_out_of_range(fixture, 0x04);
+}
+
+static void test_config_qos_expect_invalid_length(struct bt_conn *conn, uint8_t ase_id,
+						  const struct bt_gatt_attr *ase_cp,
+						  struct bt_bap_stream *stream,
+						  const uint8_t *buf, size_t len)
+{
+	const uint8_t data_expected[] = {
+		0x02,           /* Opcode */
+		0xFF,           /* Number_of_ASEs */
+		0x00,           /* ASE_ID[0] */
+		0x02,           /* Response_Code[0] = Invalid Length */
+		0x00,           /* Reason[0] */
+	};
+
+	test_preamble_state_codec_configured(conn, ase_id, stream);
+
+	ase_cp->write(conn, ase_cp, buf, len, 0, 0);
+
+	expect_bt_gatt_notify_cb_called_once(conn, BT_UUID_ASCS_ASE_CP, ase_cp, data_expected,
+					     sizeof(data_expected));
+}
+
+ZTEST_F(test_ase_control_params, test_config_qos_number_of_ases_0x00)
+{
+	uint8_t ase_id = test_ase_id_get(fixture->ase);
+	const uint8_t buf[] = {
+		0x02,                   /* Opcode = Config QoS */
+		0x00,                   /* Number_of_ASEs */
+		ase_id,                 /* ASE_ID[0] */
+		0x01,                   /* CIG_ID[0] */
+		0x01,                   /* CIS_ID[0] */
+		0xFF, 0x00, 0x00,       /* SDU_Interval[0] */
+		0x00,                   /* Framing[0] */
+		0x02,                   /* PHY[0] */
+		0x64, 0x00,             /* Max_SDU[0] */
+		0x02,                   /* Retransmission_Number[0] */
+		0x0A, 0x00,             /* Max_Transport_Latency[0] */
+		0x40, 0x9C, 0x00,       /* Presentation_Delay[0] */
+	};
+
+	test_config_qos_expect_invalid_length(&fixture->conn, ase_id, fixture->ase_cp,
+					      &fixture->stream, buf, sizeof(buf));
+}
+
+ZTEST_F(test_ase_control_params, test_config_qos_too_short)
+{
+	uint8_t ase_id = test_ase_id_get(fixture->ase);
+	const uint8_t buf[] = {
```

Comment:
Since `buf` is `const`, should `ase_id` be `const` too? 

It doesn't really affect anything in any case, but it's nice to be consistent